microservice/challenge-1/ReminderAppl/Docker-Compose/docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+version: "3.3"
+
+services:
+
+  reminder-app-local:
+    build:
+      context: ../
+      dockerfile: Dockerfile
+      target: dev
+    container_name: reminder-app-local
+    env_file: ./.env
```

Comment:
the checked in env file is call .dev.env not .env

Reply:
Updated